README.md

```
@@ -16,52 +16,59 @@
+- POSIX-compliant OS (SysV ABI) (used: Arch Linux 6.16.9-arch1-1)
+- meson >= 1.4.0 (used: meson 1.9.1)
+- cmake >= 3.20 (used: cmake 4.1.1)
+- ninja >= 1.11 (used: ninja 1.21.1)
+- clang/++ >= 18.0.0 (used: 20.1.8)
+- nasm >= 2.16 (used: 2.16.03)
+- qemu >= 7.0.0 (used: 10.1.0)
+- mkfs.fat
 - mkfs.ext2
-- grub-mkrescue
+- grub-mkrescue (used: 2:2.12.r359.g19c698d12-1)
     - Note: depending on your distro, grub-mkrescue may require xorriso and mtools packages.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)